justfile

```diff
@@ -7,10 +7,21 @@ export DEBUG := env_var_or_default('DEBUG', "")
 lint:
   shellcheck -x */*.sh jobrunner/bashrc
 
-build:
-  make test-image keys/testuser
+build: testuser-key build-test-image
 
-test:
+build-test-image:
+  #!/usr/bin/env bash
+  test packages.txt -ot .test-image -a core-packages.txt -ot .test-image -a purge-packages.txt -ot .test-image -a build-lxd-image.sh -ot .test-image && exit 0
```

Comment:
I guess it does because we don't have -e on? But -e might be a good thing.

In other just files I've basically done `test ! $THING || exit 0`

Which is always guaranteed to either return a 0 returncode, or exit 0

Reply:
good shout, yeah, it works because we don't have `-e` on, let me do this the other way round (& turn on `-e`...)